src/shared/filesystem.ts

```diff
@@ -18,7 +20,52 @@ export interface Stats extends fs.Stats {
 // functions
 export const access = promisify(fs.access)
 
-export const mkdir = promisify(fs.mkdir)
+const _mkdir = promisify(fs.mkdir)
+interface ErrorWithCode {
+    code?: string
+}
+
+export async function mkdir(
+    path: PathLike,
+    options?: number | string | MakeDirectoryOptions | undefined | null,
+): Promise<void> {
+    try {
+        await _mkdir(path, options)
+    } catch (err) {
+        // mkdir calls with recurse do not work as expected when called through electron.
```

Comment:
Can you add a note that this will be be fixed if/when VS Code adopts Electron 5? We should monitor the issue and remove this workaround when it is resolved.

Reply:
Definitely -- the provided link isn't intended for tracking, only for background info into the situation